components/post-card-header/PostCardHeader.tsx

```diff
@@ -30,11 +30,18 @@ export const PostCardHeader = ({
 }) => {
   const { lastName, userName, firstName, identifier, avatarUrl } =
     useUserInfo();
-  const router = useRouter();
+  const { isBpMDDown } = useBreakpoints();
+
+  const avatarPosition = useMemo(() => {
```

Comment:
could we rename this variable, so it would be clear that it is a boolean value? e.g. hasAvatarPosition or isAvatarPositioned etc.